src/addLiquidity.ts

```diff
@@ -109,19 +122,14 @@ export class LiquidityAddition {
         BigInt(params.precision),
       );
       amplifier = Number(dAmplifier) / (this.pool.internalState.PRECISION ?? 1);
+      txFee = getTxFee(swapCalc.mintTokensInvariantIterations, 3);
```

Comment:
ok, I might be missing something. I thought we needed 1 extra transaction instead of 3